coreneuron/sim/fadvance_core.cpp

```diff
@@ -78,11 +78,11 @@ void dt2thread(double adt) { /* copied from nrnoc/fadvance.c */
             } else {
                 nt->cj = 1.0 / dt;
             }
-            // clang-format off
-
-            #pragma acc update device(nt->_t, nt->_dt, nt->cj) \
-                    async(nt->stream_id) if(nt->compute_gpu)
-            // clang-format on
+            nrn_pragma_acc(update device(nt->_t, nt->_dt, nt->cj)
+                               async(nt->stream_id) if (nt->compute_gpu))
+            nrn_pragma_omp(target update to(nt->_t, nt->_dt, nt->cj) depend(inout
+                                                                            : nt)
+                               nowait if (nt->compute_gpu))
```

Comment:
nitpick: we should keep clang-format and make these pragmas more readable.

Reply:
I left this as it is. we will revisit when we will enable clang-format CI check.

Reply:
I fixed up some of the worst cases.